samples/sensor/stm32_vbat_sensor/boards/b_u585i_iot02a.overlay

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2022 STMicroelectronics
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Application overlay for creating vbat sensor device instance
+ */
+
+/ {
+	stm-vbat {
+		compatible = "st,stm32-vbat";
+		ratio = <4>;
+		io-channels = <&adc4 14>;
+		status = "okay";
+	};
+};
+
+&adc4 {
+	has-vbat-channel;
+};
```

Comment:
I think this should be put directly in stm32u5xxx.dtsi file

Reply:
done
Note that samples/sensor/stm32_vbat_sensor/boards/nucleo_wb55rg.overlay is not changed in that PR but should also be updated in the same way